realm/realm-library/src/main/cpp/io_realm_internal_Property.cpp

```diff
@@ -63,7 +63,7 @@ JNIEXPORT jlong JNICALL Java_io_realm_Property_nativeCreateProperty__Ljava_lang_
     return 0;
 }
 
-JNIEXPORT void JNICALL Java_io_realm_Property_nativeClose(JNIEnv* env, jclass, jlong property_ptr)
+JNIEXPORT void JNICALL Java_io_realm_internal_Property_nativeClose(JNIEnv* env, jclass, jlong property_ptr)
```

Comment:
Uh, sorry, i totally misunderstood your comments!! I can move it to phantom refs in this PR as well!! Surely can do!!